src/MCPServer/lib/assets/workflow.json

```diff
@@ -902,7 +902,35 @@
                 "replacements": [
                     {
                         "description": {
-                            "en": "5 - normal compression mode",
+                            "en": "1 - fastest compression",
+                            "es": "1 - el modo más rápido",
+                            "fr": "1 - mode rapide",
+                            "ja": "1 - 最高速モード",
+                            "pt_BR": "1 - modo mais rápido",
+                            "sv": "1 - snabbaste läget"
+                        },
+                        "id": "ecfad581-b007-4612-a0e0-fcc551f4057f",
+                        "items": {
+                            "AIPCompressionLevel": "1"
+                        }
+                    },
+                    {
+                        "description": {
+                            "en": "3 - fast compression",
+                            "es": "3 - mode de compresión rápido",
+                            "fr": "3 - mode de compression rapide",
+                            "ja": "3 - 高速圧縮モード",
+                            "pt_BR": "3 - modo de compressão rápida",
+                            "sv": "3 - snabbkomprimeringsläge"
+                        },
+                        "id": "85b2243e-ff97-4ca8-80e8-3c6b0842b360",
+                        "items": {
+                            "AIPCompressionLevel": "3"
+                        }
+                    },
+                    {
+                        "description": {
+                            "en": "5 - normal compression",
```

Comment:
Unfortunately changing the option name will break AMAUAT so I feel there are two options before merging, 

* We revert the name to: `5 - normal compression mode`.
* Create a PR submitted against AMAUAT to look for the new string [here](https://github.com/artefactual-labs/archivematica-acceptance-tests/blob/14bcaef70793668d43071083ee00521077b9b862/amuser/am_browser_ability.py#L477-L480).

I have no opinion about which is the correct approach, but I'd be tempted to keep the scope of this issue to re-ordering which doesn't impact AMAUAT, and then we can address renaming another time. If there's the capacity, a PR against AMAUAT would be equally welcome now.

Reply:
@ross-spencer Is that the only one? I've renamed other workflow choices as well, so I'd prefer to keep those in and change the AMAUAT.

Reply:
I believe it might be having a look today. So that's not so bad. The change needed to be made [here](https://github.com/artefactual-labs/archivematica-acceptance-tests/blob/14bcaef70793668d43071083ee00521077b9b862/amuser/am_browser_ability.py#L477-L480) (for future reference) and I've created a PR while I was in there so you can have a look [here](https://github.com/artefactual-labs/archivematica-acceptance-tests/pull/154). If y'all rebase and merge this during the next sprint, and then Douglas or yourself can review that change, and it will quickly show up if that's not the only problem when we run the integration tests in Jenkins again. It's just something to consider for future workflow changes.